### DISCUSSION TITLE
Builds for ARM32/64

### DISCUSSION THREAD

**Discussion by NotJustPizza** [timvisee/lazymc#6] [Ideas]
It would be great to include builds for ARM servers & microcomputers in future releases. I know it's possible to build project locally from source (I managed to build it on RPI 3B+), but it requires a lot of dependencies and shrinks typically limited disk space on ARM devices.

I already experimented with Rust target for `aarch64-unknown-linux-gnu`, but I lack experience with cross-compilation in Rust, so I couldn't make it work.

**Comment by timvisee** [comment]
Good news! I've just set this up, both for `armv7-unknown-linux-gnueabihf` and `aarch64-unknown-linux-gnu`.

The next release should [automatically](https://github.com/timvisee/lazymc/blob/261acafab0bb9b2eebd97e4aa97cde1c73e7d1d7/.gitlab-ci.yml#L109-L177) include these binaries.

I haven't extensively tested this yet though, because I don't have ARM hardware available right now.

You can find some builds here:  
- ARMv7: https://gitlab.com/timvisee/lazymc/-/jobs/1782737666/artifacts/browse
- aarch64: https://gitlab.com/timvisee/lazymc/-/jobs/1782737667/artifacts/browse
- Latest master (artifacts button on the right): https://gitlab.com/timvisee/lazymc/-/pipelines?scope=branches&page=1

**Reply by timvisee** [reply]
I would like to add that the latest release now includes these ARM builds: https://github.com/timvisee/lazymc/releases/latest
